esda/losh.py

```diff
@@ -132,3 +143,19 @@ def _statistic(y, w, a):
                 ((n*squared_rowsum) - (rowsum**2))
 
         return (Hi, ylag, yresid, VarHi)
+    def _statistic_bootstrap(y,w,a,m):
```

Comment:
```suggestion

    def _statistic_bootstrap(y,w,a,m):
```

Can you make sure it follows the `black` formatting guidelines? (I am not sure we actually check linting within CI here.)